thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/AnomalyFunctionDTO.java

```diff
@@ -70,48 +68,10 @@ public boolean isToCalculateGlobalMetric() {
    * @return a Properties object corresponds to the properties String of this anomaly function.
    */
   public Properties toProperties() {
-    Properties props = null;
-    try {
-      props = toProperties(getProperties());
-    } catch (IOException e) {
-      LOGGER.warn("Failed to parse property string ({}) for anomaly function: {}", getProperties(), getId());
-
-    }
+    Properties props = StringUtils.decodeCompactedProperties(getProperties());
```

Comment:
Please enclose in try catch

Reply:
Why do we want to catch the exception from decodeCompactedProperties? If the exception is caught, what is the expected behavior if there is an exception?

Reply:
Fixed